s3_file_uploads/fields.py

```diff
@@ -0,0 +1,11 @@
+from rest_framework import serializers
+
+from s3_file_uploads.models import UploadedFile
+
+
+class UploadedFilePrimaryKeyRelatedField(serializers.PrimaryKeyRelatedField):
```

Comment:
As this is only used for tests, it should probably live in the `/tests` folder

Reply:
This is actually a field utility that can be used in the app - the test is too actually test this